### ISSUE TITLE
Colab documentation + tutorial

### ISSUE THREAD

**Issue by jasmine-schoch** [VirtualBrainLab/UnityNeuroscience#92]
Initial notebook:
- [x] link data directly
- [x] build notebook to run locally
- [x] ensure everything runs seamlessly from one place
- [x] #118 

General next couple steps:
- [x]  move all basics tutorials onto colab
- [x] incorporate real data  #119 
- [x] #123
- [x] *** #127
- [x] #133
- [x] #125
- [x] #132
- [x] #128
- [x] #137

once colab branch is ready to be merged with main, colab html links need to be modified **

**Comment by jasmine-schoch** [comment]
General next couple of steps:
- [ ] move all basics tutorials onto colab
- [ ] incorporate real data
- [ ] upload data to use
- [ ] create links to colab within documentation
- [ ] create pre-rendered notebook

**Comment by jasmine-schoch** [comment]
Once colab branch is merged into main, all colab html links will have to be uploaded

**Comment by jasmine-schoch** [comment]
[Adding colab buttons to all basics](https://github.com/VirtualBrainLab/urchin-examples/commit/81197ec7fd6f07616ae11d6694830d877144f4df)

note: tutorials have not been tested, some still incomplete

**Comment by jasmine-schoch** [comment]
81197ec7fd6f07616ae11d6694830d877144f4df

**Comment by jasmine-schoch** [comment]
81197ec7fd6f07616ae11d6694830d877144f4df

**Comment by jasmine-schoch** [comment]
add probe examples: https://github.com/VirtualBrainLab/Urchin/issues/94
